lib/eex/lib/eex/compiler.ex

```diff
@@ -14,9 +14,9 @@ defmodule EEx.Compiler do
     file = opts[:file] || "nofile"
     line = opts[:line] || 1
     column = 1
-    indent = opts[:indent] || 0
+    indentation = opts[:indentation] || 0
     trim = opts[:trim] || false
-    tokenizer_options = [trim: trim, indent: indent]
+    tokenizer_options = [trim: trim, indentation: indentation]
```

Comment:
Let's make it a map and pass it to the tokenizer. The tokenizer should always assume a proper map is given (it may require some changes in the tests but since it is private API, it should be fine).